content/docs/gke/deploy/monitor-iap-setup.md

```diff
@@ -102,28 +121,33 @@ While it requires some effort, the end result is well worth it
           type: ValidateFailed
       ```
 
-     * The most recent condition should be **Certificate issued successfully**
-     * It can take around 10 minutes to provision a certificate after the GCP loadbalancer is created
-     * The most common error is hitting [Let's Encrypt quota issues](https://letsencrypt.org/docs/rate-limits/)
+    It can take around 10 minutes to provision a certificate after the 
+    creation of the load balancer.
+
+    The most recent condition should be `Certificate issued successfully`.
 
-       * Let's Encrypt enforces a quota of 5 duplicate certificates per week 
-       
-       * The easiest fix to quota issues is to pick a different hostname by recreating and redeploying Kubeflow with a different
-         name 
+    The most common error is running out of [Let's Encrypt 
```

Comment:
This is the second  instance of "the most common error is ____" in this document. It might be better to just say that "A common error for Let's Encrypt is to run out of certificate quota."

Perhaps also apply the same to the previous instance of "the most common error", which is about GCP quota. It might be good to help distinguish the two quota types more clearly.